src/data/tournament.py

```diff
@@ -962,18 +967,21 @@ def build_boards(
                     key=attrgetter('rating'),
                     reverse=True,
                 )
-                weak_time = self.time_control_initial_time
+                weak_time = self.time_control_initial_time or 0
                 rating_diff = strong_player.rating - weak_player.rating
-                penalties = rating_diff // self.time_control_handicap_penalty_step
+                penalties = rating_diff // (
+                    self.time_control_handicap_penalty_step or 0
```

Comment:
not a good idea, it will raise a `ZeroDivisionError` if the code path is somehow executed with a `None` value.
The best case would be to separate the cases and set `penalties = 0` if `self.time_control_handicap_step is None`